### PR TITLE
Multiples and Langton's Ant katas tasks

### PR DESCRIPTION
**Pull request by cjflores1**

### CODE REVIEW

File: katas/langtonant/charles/index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```
Comment (J0c4) [inline katas/langtonant/charles/index.html:1]: Hi Charles, please update your code so it doesn't need to use an html page, it should be enough to print in console the final state of the grid after ant takes the given number of steps

**Review by J0c4 — CHANGES_REQUESTED** [review]

File: katas/langtonant/charles/index.html (hunk 1 above)
Comment (cjflores1) [reply katas/langtonant/charles/index.html:1]: Ok, I'll do it. Thanks

File: katas/langtonant/charles/langtonant.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        for (let indexColumn = 0; indexColumn < arg1; indexColumn++) {
+            gridArray[indexRow][indexColumn] = (indexRow == startPosition && indexColumn == startPosition) ? startStepDirection : '   ';
+        }
+    }
+}
+
+function directionNextStep(rowPosition, columnPosition) {
+    switch (gridArray[rowPosition][columnPosition]) {
+    case directionUp:
+        pointer = rowPosition - 1;
```
Comment (J0c4) [inline katas/langtonant/charles/langtonant.js:31]: For each case, we have some repeated code that could be improved by adding a function that parameterizes the values that change on every situation, consider that for your future implementations

**Review by J0c4 — APPROVED** [review]
Nice implementation

**Pull request merged**